code/game/gamemodes/bloodbrother/brother.dm

```diff
@@ -0,0 +1,174 @@
+/datum/antagonist/brother
+	name = "Brother"
+	roundend_category = "Brother"
+	job_rank = ROLE_BROTHER
+	var/special_role = ROLE_BROTHER
+	var/give_objectives = TRUE
+	var/datum/team/brother_team/team
+
+/datum/antagonist/brother/create_team(datum/team/brother_team/new_team)
+	if(!new_team)
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization")
+	team = new_team
+
+/datum/antagonist/brother/get_team()
+	return team
+
+/datum/antagonist/brother/on_gain()
+	SSticker.mode.brothers += owner
+	owner.objectives += team.objectives
+	owner.special_role = special_role
```

Comment:
Is there a case where `special_role` can be something other than `ROLE_BROTHER`? If not you can probably just use the define directly here.